.github/workflows/pscale-pr.yml

```
@@ -0,0 +1,89 @@
+          PW=$(pscale password create "$PSCALE_DB" "$BRANCH" "ci-${GITHUB_SHA::8}" \
+                 --org "$PSCALE_ORG" --format json)
+          USER=$(echo "$PW" | jq -r .username)
+          PASS=$(echo "$PW" | jq -r .plain_text)
+          HOST=$(echo "$PW" | jq -r .access_host_url)
+          export DRIZZLE_DATABASE_URL="mysql://$USER:$PASS@$HOST/$PSCALE_DB?ssl={}"
+          pnpm --dir=web/internal/db run migrate
+
+      - name: ensure deploy request
+        # Open a DR into staging so reviewers have a one-click promotion
+        # surface. Reuse existing open DR for repeat runs of the same PR.
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)